cmd/cluster_phase_etcd.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/xetys/hetzner-kube/cmd/phases"
+	"github.com/xetys/hetzner-kube/pkg"
+)
+
+func init() {
+	command := declarePhaseCommand("etcd", "setups a etcd cluster", func(cmd *cobra.Command, args []string) {
```

Comment:
I know that is a good idea be DRY but IMHO we should declare the command without helper like `declarePhaseCommand` and moving function outside init (imho should be awesome if we are able to move all this stuff in a package that can be invoked from the cmd part of the application)